lib/LedgerSMB/PSGI.pm

```diff
@@ -14,6 +14,10 @@ LedgerSMB::PSGI - PSGI application routines for LedgerSMB
 use strict;
 use warnings;
 
+use PGObject;
+
+BEGIN { PGObject->new_registry('ledgersmb') }
```

Comment:
but for 1.6 I don't see a problem.  Will create a 2.1 today with this feature and then amend.